assets/scss/_variables_project.scss

```diff
@@ -488,20 +560,31 @@ ol.breadcrumb li:not(:first-child):before {
 
 // Table of Contents Component
 #TableOfContents {
-    
+    font-family: $base-font;
     font-size: 13px !important;
-    font-family: "Source Sans Pro", sans-serif;
     a{
         margin: 0px !important;
         padding: 8px !important;
         font-weight: 500 !important;
+        font-size: 13px !important;
```

Comment:
I wonder if this should also be 12 to align with the breadcrumbs, which are also a navigation element. Also not sure you need line 564 and 569 both. It would be bad if they went out of sync.